### DISCUSSION TITLE
System Update information

### DISCUSSION THREAD

**Discussion by JanOlsmar** [oqtane/oqtane.framework#1950] [Q&A] [answered]
When a new release is out as now 3.0.2 i expext  the information to show up there. But it tells me the framework ia already up to Date.
Have I missed some setting or is this function not live yet?

ps. I love Oqtane and have learned a lot just reading the code. Thanks. ds.

**Comment by sbwalker** [comment, marked as answer]
This turned out to be a configuration issue with the service - it is working as expected now

**Reply by JanOlsmar** [reply]
Yes  worked nice to upgrade to 3.0.3 Thanks.

**Comment by sbwalker** [comment]
The process appears to be seamless from a user perspective but there are actually a lot of moving parts behind the scenes...

![image](https://user-images.githubusercontent.com/4840590/154297101-5f0e90c5-15f1-4469-9f84-4bd978dbb962.png)
